doc_source/aws-resource-ses-receiptruleset.md

```diff
@@ -45,6 +46,14 @@ The name of the rule set to create\. The name must:
  *Type*: String  
  *Update requires*: [Replacement](using-cfn-updating-stacks-update-behaviors.md#update-replacement) 
 
+## Return Values<a name="aws-resource-ses-receiptruleset-returnvalues">
+
+## Ref
+
+When the logical ID of this resource is provided to the `Ref` intrinsic function, `Ref` returns the ID of the ReceiptRuleSet resource \(the physical resource ID\)\.
```

Comment:
Instead of "returns the ID of the ReceiptRuleSet," will you please say "returns the name of the ReceiptRuleSet"?
Also, please don't include "(the physical resource ID)" because "name" is more specific.